source/deployment/bulk-loading-data-format.rst

```diff
@@ -490,10 +474,120 @@ Fields of the User object
       <td align="center" valign="middle">No</td>
       <td align="center" valign="middle">No</td>
     </tr>
+    <tr class="row-odd">
+      <td valign="middle">tutorial_step</td>
+      <td valign="middle">string</td>
+      <td>Where to start the user tutorial. Must be one of the following values:<br>
+        <kbd>"1"</kbd>, <kbd>"2"</kbd> or <kbd>"3"</kbd> - Start from the specified tutorial step.<br>
+        <kbd>"999"</kbd> - Skip the user tutorial.</td>
+      <td align="center" valign="middle">No</td>
+      <td align="center" valign="middle">No</td>
+    </tr>
   </table>
 
-Fields of the TeamMembership object
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Fields of the UserNotifyProps object
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This object is a member of the User object.
+
+.. raw:: html
+
+  <table width="100%" border="1" cellpadding="5px" style="margin-bottom:20px;">
+    <tfoot><tr><td colspan="5">[1] Not validated, but an error occurs if no such team exists when running in apply mode.</td></tr></tfoot>
+    <tr class="row-odd">
+      <th class="head">Field name</th>
+      <th class="head">Type</th>
+      <th class="head">Description</th>
+      <th class="head">Validated</th>
+      <th class="head">Mandatory</th>
+    </tr>
+    <tr class="row-odd">
+      <td valign="middle">desktop</td>
+      <td valign="middle">string</td>
+      <td>Preference for sending desktop notifications. Must be one of the following values:<br>
+      <kbd>"all"</kbd> - For all activity.<br>
+      <kbd>"mention"</kbd> - Only for mentions.<br>
+      <kbd>"none"</kbd> - Never.</td>
+      <td align="center" valign="middle">No</td>
+      <td align="center" valign="middle">No</td>
+    </tr>
+    <tr class="row-odd">
+      <td valign="middle">desktop_duration</td>
+      <td valign="middle">string</td>
+      <td>Preference for how long desktop notifications remain on screen. Must be one of the following values:<br>
+      <kbd>"3"</kbd> - 3 seconds.<br>
+      <kbd>"5"</kbd> - 5 seconds.<br>
+      <kbd>"10"</kbd> - 10 seconds.<br>
+      <kbd>"unlimited"</kbd> - Unlimited.</td>
+      <td align="center" valign="middle">No</td>
+      <td align="center" valign="middle">No</td>
+    </tr>
+    <tr class="row-odd">
+      <td valign="middle">desktop_sound</td>
+      <td valign="middle">string</td>
+      <td>Preference for whether desktop notification sound is played. Must be one of the following values:<br>
+      <kbd>"on"</kbd> - Sound is played.<br>
+      <kbd>"off"</kbd> - Sound is not played.</td>
```

Comment:
The values for these are "true" and "false" but note that those are strings containing the value "true" or "false", not booleans.